app/script/util/Environment.js

```diff
@@ -46,39 +46,39 @@ window.z.util = z.util || {};
   };
 
   const _check = {
-    get_version: () => {
+    getVersion: () => {
       if (window.platform.version) {
         return window.parseInt(window.platform.version.split('.')[0], 10);
       }
     },
-    is_chrome() {
-      return window.platform.name === BROWSER_NAME.CHROME || this.is_electron();
+    isChrome() {
+      return window.platform.name === BROWSER_NAME.CHROME || this.isElectron();
```

Comment:
`isChrome: () => window.platform.name === BROWSER_NAME.CHROME || this.isElectron(),`